src/openpa/src/opa_primitives.h

```diff
@@ -83,6 +83,8 @@
 #if defined(OPA_USE_UNSAFE_PRIMITIVES)
 /* comes first to permit user overrides in the style of NDEBUG */
 #include "primitives/opa_unsafe.h"
+#elif defined(OPA_HAVE_GCC_INTRINSIC_ATOMICS)
+#include "primitives/opa_gcc_intrinsics.h"
```

Comment:
Is there any performance data we can look at for this change? I would really like to drop the assembly, too, and my understanding was that moving to MPL atomics would do so. Doing so with OPA and without memory model aware atomics may be premature.

Reply:
I probably should remove this commit as it turns out not helping much. Originally I thought gcc intrinsics would compile cleanly, but turns out it emits tons of warning as well. In the end I just turned off both warnings using `#pragma`.